settings/pkg/config/defaults/defaultconfig.go

```diff
@@ -44,13 +46,22 @@ func DefaultConfig() *config.Config {
 			Addr:      "127.0.0.1:9191",
 			Namespace: "com.owncloud.api",
 		},
-		DataPath: path.Join(defaults.BaseDataPath(), "settings"),
+		StoreType: "metadata", // use metadata or filesystem
+		DataPath:  path.Join(defaults.BaseDataPath(), "settings"),
 		Asset: config.Asset{
 			Path: "",
 		},
 		TokenManager: config.TokenManager{
 			JWTSecret: "Pive-Fumkiu4",
 		},
+
+		Metadata: config.Metadata{
+			GatewayAddress:    "127.0.0.1:9142",
+			StorageAddress:    "127.0.0.1:9215",
+			ServiceUserID:     "ddc2004c-0977-11eb-9d3f-a793888cd0f8",
+			ServiceUserIDP:    os.Getenv("OCIS_URL"),
```

Comment:
I really dislike the service user idp setting, but as far as I understood we cannot easily work around it. I think we should at least configure it like the other OCIS_URL / IDP URL settings. (see eghttps://github.com/owncloud/ocis/blob/51d1eb0182fce0ed3920689ac3ed5383b2eb2e84/idp/pkg/config/config.go#L56, https://github.com/owncloud/ocis/blob/51d1eb0182fce0ed3920689ac3ed5383b2eb2e84/idp/pkg/config/defaults/defaultconfig.go#L38)

Reply:
:+1: That's what I meant with "hardcoding it"